tools/pom.xml

```
@@ -19,7 +19,7 @@
                     Note: see https://github.com/unicode-org/icu/packages/411079/versions for the
                     icu4j.version tag to use
                 -->
-		<icu4j.version>68.1-cldr-2020-10-21</icu4j.version>
+		<icu4j.version>68.1-release-68-1</icu4j.version>
```

```suggestion
		<icu4j.version>68.1-release-68-1</icu4j.version>
```

See the URL on line 19 - you can see the exact tag names. If you click one such as https://github.com/unicode-org/icu/packages/411079?version=68.1-release-68-1 you can see them in detail: `<version>68.1-release-68-1</version>`